home-application/src/main/java/com/softserveinc/ita/homeproject/application/apiservice/UserApiImpl.java

```diff
@@ -1,6 +1,6 @@
 package com.softserveinc.ita.homeproject.application.apiservice;
```

Comment:
Let's rename the package to `api`. So it will be `com.softserveinc.ita.homeproject.application.api`

Reply:
Done.